portfolio.html

```diff
@@ -6,21 +6,25 @@
 		<title>My Portfolio</title>
 		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
 	</head>	
+	
 	<body>
+		
 		<nav class="nav_bar">
 			<div><label class="label_karen_schulz" for="nav_bar">Karen Schulz</label></div>
 			<div><a href="#">Home</a>
 			<a href="#">Contact</a></div>
 		</nav>
-		<div class="welcome"><img class="mypicture" src="images/profile_pic.png" alt="my_profile_picture">
-			<section>
-			<div class="heading"><h2>Welcome to my Portfolio</h2></div>
-			<div class="portfolio1"><img class="first_work" src="images/green_back.png" alt="my_first_work"><p>I love to do illustrations, check my creations</p>
-			<div class="portfolio2"><img class="second_work" src="images/eating.png" alt="my_second_work"><p>I love html and CSS</p>
-			<div class="portfolio3"><img class="third_work" src="images/full-pouch.png" alt="my_first_work"><p>I've written two children's books</p>>
-		<div><a href="#">Home</a>
-		<a href="#">Contact</a></div>
-	</section>
+		
+		<div class="portfolio_page">
+			<p><img class="mypicture" src="images/profile_pic.png" alt="my_profile_picture"></p>
+		
+			<div class="heading"><h1>Welcome to my Portfolio</h1></div>
+			<div class="portfolio1"><p><img class="first_work" src="images/green_back.png" alt="my_first_work"></p><p>I love to do illustrations, check my creations</p></div>
+			<div class="portfolio2"><img class="second_work" src="images/eating.png" alt="my_second_work"><p>I love html and CSS</p></div>
+			<div class="portfolio3"><img class="third_work" src="images/full-pouch.png" alt="my_first_work"><p>I've written two children's books</p></div>
+		<div><p><a href="#">Home</a></p>
+		<p><a href="#">Contact</a></div></p>
+		
 	</body>
 	
 	<footer>C Copyrights. All rights reserved. Karen Schulz, January 2021</footer>
```

Comment:
I think the <footer> tags need to be within the body tag

Reply:
hi, do you mean the footer needs to be inside the body tag?

Reply:
I corrected that. Thanks!